index.bs

```diff
@@ -168,15 +168,16 @@ The <dfn method for=FileSystemHandle>requestPermission(|descriptor|)</dfn> metho
 ## The {{FileSystemFileHandle}} interface ## {#api-filesystemfilehandle}
 
 <xmp class=idl>
-dictionary FileSystemCreateWriterOptions {
+dictionary FileSystemCreateWritableOptions {
+  unsigned long long position = 0;
```

Comment:
I think I'd prefer having this be something that is passed to getWriter, rather than have it be a part of creating the stream. I.e. have FileSystemWritableStream.getWriter take an (optional?) position that starts the writer off at that position. This really is a very different option than the other options that can be passed when creating a writer thing, and it also seems more consistent to me if the position is something that you pass in to all the methods in FileSystemWritableStream, rather than have some use some kind of implied position and some have an explicit position.

Having said that, I guess we'd still need some kind of implicit default position for when no offset is specified for getWriter, or when you pipe another stream into this stream... In that case I think having a seek() on the stream itself would still be better than passing in a position here.

But curious what domenic thinks.

Reply:
I personally prefer `seek()` on the stream as it's more explicit. However, we still have the issue of the implicit position on the stream.

Reply:
I don't have strong feelings here. I think one could argue that this is a peer to inPlace, but I'm not sure (and inPlace doesn't seem to have a domintro for me to skim to try to understand it).

I do think you need to store a position on the stream if you make it part of getWriter(); i.e. you should finish with one writer, then getting a new one should pick up where you left off. And at that point it's not much different then adding seek(). You could even say that seek() just adjusts where the next getWriter() will start from.

If you instead make this an argument to the stream, then I could see getWriter() always starting at the specified position... Basically the stream represents "the file at position X", and you can keep overwriting that segment multiple times by getting multiple writers.

Reply:
updated to use a `seek` method.